src/app/shared/components/template/components/date-time-picker/date-time-picker.component.md

```diff
@@ -0,0 +1,27 @@
+# DateTimePicker
```

Comment:
+1 - thanks for adding the documentation, really nice that it also links through to the demos